site/docs/components/stepped-tracker/examples.mdx

```diff
@@ -0,0 +1,34 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Examples
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+<LivePreview componentName="component-name" exampleName="ComponentExample" title="Basic Features">
+
+The SteppedTracker contains multiple TrackerStep child components. Each TrackerStep indicates its status through its icon. The connectors indicate current active step progress through the process. You should add a label to the TrackerStep using the StepLabel component as follows:
```

Comment:
I think these example blurbs (and perhaps the examples themselves?) need re-writing. Unlike the StoryBook docs, where this all gets rendered as a continuous block of content, on the Salt site, individual examples can be viewed stand-alone. Therefore each example + blurb should be self-contained.

Also, examples are rendered _above_ the blurb (so "...as follows:" doesn't make sense - perhaps "...as shown in this example's code" would be better?)